server/routes/routes.js

```diff
@@ -164,6 +164,7 @@ async function findDirectRoutes(startStops, endStops, visitedStop =0, pastTripId
             departureTime: {
               gte: formattedCurrentTime,
               lte: formattedOneHourLater
+
```

Comment:
get rid of space

Reply:
Sure! This has been fixed in future commits